.github/workflows/markdown.yaml

```diff
@@ -1,11 +1,11 @@
-name: Daily Markdown Link Check
+name: Markdown
```

Comment:
I would add more context: `Markdown Link Check`

Reply:
On check's list you can see (see below):
`Markdown / link-check (pull_request) Successful in 1m`

If we name it `Markdown Link Check` it will be:
`Markdown Link Check / link-check (pull_request) Successful in 1m`

IMO it's awful repetition.